docker/all-in-one/entrypoint.sh

```diff
@@ -209,34 +209,43 @@ function start_supervisor {
   /usr/bin/supervisord -c $SUPERVISOR_CONF
 }
 
+DELEGATED_ARCHIVE_PATH=/data/delegated-init.tar.gz
+
+fetch_delegated_payload() {
+  DELEGATED_PAYLOAD_URL=$(curl -XPOST -sf "$SUPABASE_URL/$DELEGATED_INIT_LOCATION" -H 'Content-Type: application/json' -d '{"reportingToken":"'"$REPORTING_TOKEN"'"}')
+
+  if [ -z "$DELEGATED_PAYLOAD_URL" ]; then
+    echo "Failed to get response, bailing"
+    return
+  fi
+
+  curl -s --time-cond $DELEGATED_ARCHIVE_PATH -o $DELEGATED_ARCHIVE_PATH "$DELEGATED_PAYLOAD_URL"
```

Comment:
`--time-cond` here will throw some warnings if the file doesn't already exist, but they don't have any side-effects besides informational; curl also specifies in a verbose manner that it's actively disabling the flag